### DISCUSSION TITLE
use LocalMachine certificate for studio

### DISCUSSION THREAD

**Discussion by wallaceturner** [ravendb/ravendb#14691] [General]
Is it possible for all users on a server to use the certificate in the 'LocalMachine' for accessing the studio rather than having to import for each user?
we currently need to do this for each user

```
CERTUTIL -user -f -importpfx .\cert.pfx
```

if we import at the machine level (below) we dont get prompted by the browser...

```
CERTUTIL -f -importpfx .\cert.pfx
```

**Comment by garayx** [comment]
Hi,

I couldn't find a option to read the certificate from the local machine cert storage on chrome, see: https://superuser.com/questions/1718009/force-browser-to-use-local-machine-certificate-store-for-client-certificates
